src/main/java/org/jenkinsci/plugins/tokenmacro/TokenMacro.java

```diff
@@ -141,7 +144,12 @@ public boolean hasNestedContent() {
      * All registered extension points.
      */
     public static ExtensionList<TokenMacro> all() {
-        return Jenkins.getInstance().getExtensionList(TokenMacro.class);
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
+            return jenkins.getExtensionList(TokenMacro.class);
+        } else {
+            return ExtensionList.create((Jenkins) null, TokenMacro.class);
```

Comment:
This is a common pattern for extension points in Jenkins. Moreover, it has been replaced by a single method in newest Jenkins cores